integration/tests/session-tasks.test.ts

```diff
@@ -0,0 +1,35 @@
+import { appConfigs } from '../presets';
+import { testAgainstRunningApps } from '../testUtils';
+
+// TODO ORGS-566 - write integration tests for after-auth flow
+testAgainstRunningApps({ withEnv: [appConfigs.envs.withEmailCodes] })('after-auth flows @generic @nextjs', () => {
```

Comment:
This is just a placeholder for me to come back later which is helpful to visualize the redirection behavior introduced in this PR

The reason integration tests aren't being implemented is that FAPI changes for that are under development and it could break CI